src/Illuminate/Mail/Mailable.php

```diff
@@ -283,8 +283,8 @@ protected function runCallbacks($message)
     /**
      * Set the sender of the message.
      *
-     * @param  object|array|string  $address
-     * @param  string|null  $name
+     * @param  mixed  $address
```

Comment:
Nope, mixed should be avoided at all costs. It's the same as not documenting it.